src/video.tsx

```diff
@@ -188,7 +188,6 @@ const Video = React.forwardRef<VideoElement, VideoProps>(
             dispatch('finishLoad');
             if (video && shouldAutoPlay && video.paused) {
                 video.play();
-                dispatch('play');
```

Comment:
This is already done [here](https://github.com/Telefonica/mistica-web/blob/master/src/video.tsx#L223). If we do it twice, it will cause the if statement in `onTimeUpdate` callback to evaluate to false (and therefore the `onPlay` callback passed as prop won't be executed).